ladybug/stat.py

```diff
@@ -156,15 +134,22 @@ def _import_data(self):
             self._location.time_zone = time_zone
             self._location.elevation = elevation
 
-            # Pull out climate zone classifications
-            a_clim_mat = re.compile(r'Climate type\s"(\S*)"\s\(A').findall(self._body)
-            if len(a_clim_mat) > 0:
-                self._ashrae_climate_zone = a_clim_mat[0]
-            k_clim_mat = re.compile(r'Climate type\s"(\S*)"\s\(K').findall(self._body)
-            if len(k_clim_mat) > 0:
-                self._koppen_climate_zone = k_clim_mat[0]
-
-            # Pull out annual design days
+            # pull out individual properties
+            self._stand_press_at_elev = self._regex_check(
+                r"Elevation\s*[-]*\s*(\d*)Pa", self._header[5])
+            self._ashrae_climate_zone = self._regex_check(
+                r'Climate type\s"(\S*)"\s\(A', self._body, False)
+            self._koppen_climate_zone = self._regex_check(
+                r'Climate type\s"(\S*)"\s\(K', self._body, False)
+
+            # pull out extreme and seasonal weeks.
+            self._extreme_hot_week = self._regex_week_parse(
+                r"Extreme Hot Week Period selected:\s*(\S*)\s*(\S*):\s*(\S*)\s*(\S*),")
```

Comment:
We can probably be more specific in cases like this to avoid parsing errors. What you have there will also work of course.

```suggestion
                r"Extreme Hot Week Period selected:\s*(\w{3})\s*(\d{1,2}):\s*(\w{3})\s*(\d{1,2}),")
```

Reply:
Good idea.  I updated all of the regex conditions that were like this to use the same level of specificity.